packages/tools/src/eventListeners/annotations/contourSegmentation/contourSegmentationCompleted.ts

```diff
@@ -91,12 +92,12 @@ export default async function contourSegmentationCompletedListener(
   }
 }
 
-function isFreehandContourSegToolRegisteredForViewport(
+/** Indicates if this contour tool can handle combining */
+function isContourToolCombineAvailable(
```

Comment:
The goal was to check only if freehand contour segmentation is enabled because when we append/remove contour tools they get converted into freehand contour segmentation and if that tool is not available the annotation is "removed" from the viewport. This change now seems to check if the tool used to draw the annotation is available which should always return true, right?

Reply:
It still needs to check if `isFreehandContourSegToolRegisteredForViewport` is available when appending/removing contours